Remotion/Mixins/Core/Utilities/ReflectionUtility.cs

```diff
@@ -88,7 +88,7 @@ private static bool CheckMethodAttributeOnMember (MemberInfo member, MethodAttri
 
       string message = String.Format (
           "The given member {0}.{1} is neither property, method, nor event.",
-          member.DeclaringType.FullName,
+          member!.DeclaringType!.FullName,
```

Comment:
`member!` would fail if line 86, 87 is called with a null-value. For now, we should just handle it gracefully (or have TODO to hanlde it gracefully) but as a second step, the event-add/remove semantics must be analyzed / reworked for Mixins-code -> second todo task.

Reply:
I'd add a check similar to get/set above right now, if that's OK.

Reply:
Changed in b8c6cee2f4329e54cc65b7f934d94d77c7315a29.